packages/salesforcedx-vscode-core/src/commands/auth/orgLogout.ts

```diff
@@ -113,7 +113,7 @@ export async function orgLogoutDefault() {
 }
 
 async function removeUsername(username: string) {
-  await configSet('');
+  await ConfigUtil.setDefaultUsernameOrAlias('');
```

Comment:
@CristiCanizales core Config class has an unset method. I suggest creating a new function in ConfigUtil to unset target org.